scripts/upgrade

```diff
@@ -68,14 +68,20 @@ if ynh_version_gt "0.45-3" "${previous_version}" ; then
 fi
 
 if ynh_version_gt "0.77-2" "${previous_version}" ; then
-    ynh_install_nodejs 8.9.3
-	ynh_use_nodejs
+    ynh_remove_nodejs
```

Comment:
Eh will this remove the corect version of nodejs or all of them o.O"

Reply:
from https://yunohost.org/#/packaging_apps_helpers_en

`
ynh_remove_nodejs
Remove the version of node used by the app.
`

But I haven't checked the code in ynh_remove_nodejs